README.md

```diff
@@ -20,3 +20,8 @@ randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+The recursive call breaks the array in half each time so it will run at T(log n) times unless the pivot is the highest or lowest number then it would need to run through the array T(n) times. The for loop in the recursive call needs to go through every number so it would run n times. So you end up with T(nlog n) in the best case or T(n^2) in the worst case.
```

Comment:
What does $T$ mean here? And how many permutations might you have to try before finding the sorted one?